gtest-parallel

```diff
@@ -145,6 +147,9 @@ for test_binary in binaries:
     line = line.strip()
     if not options.gtest_also_run_disabled_tests and 'DISABLED' in line:
       continue
+    line = line.split('#')[0].strip()
+    if not line:
```

Comment:
I don't think so, but I think it makes sense to skip a line if it was a comment only, right?